pkg/kdump/config-client.js

```diff
@@ -47,7 +47,7 @@ export class ConfigFile {
 
     close() {
         if (this._fileHandle) {
-            this._fileHandle.remove();
+            this._fileHandle.close();
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17628-20220808-115321-9be51b56-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)